src/resolversFactory.js

```diff
@@ -1,7 +1,18 @@
 var util = require('util');
 var http = require('http');
+var peliasConfig = require( 'pelias-config' ).generate();
+
+
+function createWofPipResolver(url, config) {
+  config = config || peliasConfig;
+
+  var maxConcurrentReqs = 1;
+  if (config.imports.adminLookup && config.imports.adminLookup.maxConcurrentReqs) {
+    maxConcurrentReqs = config.imports.adminLookup.maxConcurrentReqs;
+  }
+
+  http.globalAgent.maxSockets = maxConcurrentReqs;
```

Comment:
I don't love exposing the fact that this module uses HTTP library to do the job. That's an implementation detail best abstracted away and let users of it focus on what they want done, not how. Thoughts?